### ISSUE TITLE
Invalid "Undefined variable" for keyword-only argument to lambda

### ISSUE THREAD

**Issue by anntzer** [PyCQA/pylint#1329]
### Steps to reproduce
```
$ cat /tmp/test.py && pylint --rcfile=/dev/null -dC,W /tmp/test.py
lambda *, foo: foo
************* Module test
E:  1,15: Undefined variable 'foo' (undefined-variable)

----------------------------------------------------------------------
Your code has been rated at -40.00/10 (previous run: 10.00/10, -50.00)
```

### Current behavior
Spurious error.

### Expected behavior
No spurious error.

### pylint --version output
```
No config file found, using default configuration
pylint 2.0.0, 
astroid 1.5.0
Python 3.6.0 (default, Jan 16 2017, 12:12:55) 
[GCC 6.3.1 20170109]
```

**Comment by PCManticore** [comment]
Hi, thanks for reporting this issue. Unfortunately, I cannot reproduce it, using the latest code. Here's what I'm getting:

```
C:  1, 0: Missing module docstring (missing-docstring)
W:  1, 0: Statement seems to have no effect (pointless-statement)
```

Can you test with the latest code and let me know if it works for you? Thank you!

**Comment by anntzer** [comment]
I can reproduce as of master.  Please reopen.
